### PR TITLE
Doc fixes

### PR DESCRIPTION
**Pull request by jaeilepp**
Just a set of fixes to broken stuff on the website.
Closes https://github.com/mne-tools/mne-python/issues/3088

### CODE REVIEW

File: doc/manual/source_localization/forward.rst [hunk 1]
```diff
@@ -180,7 +180,7 @@ and
 
 .. tabularcolumns:: |p{0.2\linewidth}|p{0.3\linewidth}|p{0.5\linewidth}|
 .. _CHDJDEDJ:
-.. table:: Coordinate transformations in FreeSurfer and MNE software packages. The symbols :math:`T_x` are defined in :ref:`CHDFFJIJ`. Note: mne_make_cor_set /mne_setup_mri prior to release 2.6 did not include transformations :math:`T_3`, :math:`T_4`, :math:`T_-`, and :math:`T_+` in the fif files produced.
+.. table:: Coordinate transformations in FreeSurfer and MNE software packages.
```
Comment (jaeilepp) [inline doc/manual/source_localization/forward.rst]: The links on table headers don't seem to work very well (https://circle-artifacts.com/gh/mne-tools/mne-python/621/artifacts/0/home/ubuntu/mne-python/doc/_build/html/manual/source_localization/forward.html#chdjdedj)
Also the links seem wrong with these long titles. See if you agree with the changes once the circle is ready.

File: mne/__init__.py [hunk 2]
```diff
@@ -80,6 +80,7 @@
 from .selection import read_selection
 from .dipole import read_dipole, Dipole, DipoleFixed, fit_dipole
 from .channels import equalize_channels, rename_channels, find_layout
+from .report import Report
```
Comment (jaeilepp) [inline mne/__init__.py]: For some reason ``Report`` was not added to the namespace.

**Comment by agramfort** [comment]
thanks @jaeilepp !

merge if you're happy and circle reports good rendering

if you can do a pass on tutorials it would be great.

cf mailing list I realize that there is no tutorial on averaging ERP/ERF...

we also need a tutorial on visualization of Evoked and Epochs data

File: doc/install_mne_c.rst [hunk 3]
```diff
@@ -129,7 +129,7 @@ the three commands:
 
     $ export MNE_ROOT=<MNE>
     $ export MATLAB_ROOT=<Matlab>
-    $ $MNE_ROOT/bin/mne_setup_sh
+    $ . $MNE_ROOT/bin/mne_setup_sh
```
Comment (agramfort) [inline doc/install_mne_c.rst]: I think you miss some. See below

File: doc/manual/cookbook.rst [hunk 4]
```diff
@@ -274,13 +274,13 @@ segmentation.
...
 Using this model, the BEM solution can be computed using
-:func:`mne.make_bem_solution`` as::
+:func:`mne.make_bem_solution` as::
 
     >>> bem_sol = make_bem_solution(model)
     >>> write_bem_solution('sample-5120-5120-5120-bem-sol.fif', bem_sol)
 
 After the BEM is set up it is advisable to check that the
-BEM model meshes are correctly positioned using *e.g.*, :class:`mne.Report`.
+BEM model meshes are correctly positioned using *e.g.*, :class:`mne.report.Report`.
```
Comment (Eric89GXL) [inline doc/manual/cookbook.rst]: +1 for making this a top-level class instead

Comment (Eric89GXL) [inline doc/manual/cookbook.rst]: It looks like you've done it already below, so you should revert this line's change

File: tutorials/plot_creating_data_structures.py [hunk 5]
```diff
@@ -76,9 +76,9 @@
 # ---------------------------------------------
 #
 # To create a :class:`mne.io.Raw` object from scratch, you can use the
-# :class:`mne.RawArray` class, which implements raw data that is backed by a
+# :class:`mne.io.RawArray` class, which implements raw data that is backed by a
 # numpy array.  Its constructor simply takes the data matrix and
-# :class:`mne.io.meas_info.Info` object:
+# :class:`mne.io.Info` object:
```
Comment (Eric89GXL) [inline tutorials/plot_creating_data_structures.py]: Maybe this one should actually be top-level, too. It's not really *only* for raw IO, which is how we've been conceptualizing the `mne.io` namespace.

Comment (Eric89GXL) [inline tutorials/plot_creating_data_structures.py]: (but that could be a different PR since we say `io.Info` a lot in docstrings)

File: doc/install_mne_c.rst (hunk 3 above)
Comment (jaeilepp) [inline doc/install_mne_c.rst]: I checked on the original software manual and the dot is only used here.

**Comment by agramfort** [comment]
thx @jaeilepp 

go circle go !
